helpers/copy_tree.sh

```diff
@@ -46,6 +62,9 @@ sed -i "/$token/,\$d" "$modify_spec"
 echo "$token" >> "$modify_spec"
 echo "%post" >> "$modify_spec"
 
+BUILD_HOST_REPLACEMENT=$(grep "%define build_host_replacement" "$modify_spec" | cut -d ' ' -f3)
+BUILD_USER_REPLACEMENT=$(grep "%define build_user_replacement" "$modify_spec" | cut -d ' ' -f3)
+
```

Comment:
Let's make [this](https://github.com/mer-hybris/droid-system-device/pull/22/files#diff-491d059e29b8e3d61f1fd32e89ae18b2e3a499f1a7f5fa6e34506d73e8f510e1R6) fool-proof:)
```suggestion

if [ -n "$BUILD_HOST_REPLACEMENT" ] && [[ ${trees[0]} != *system ]]; then
    echo Make sure to pass .../system argument first to this script, this is needed for
    echo removing the ro.build.host and ro.build.user properly from .../vendor.
    exit    
fi

```

Reply:
Is it worth handling possible final slashes (e.g. ~/system/) in this?